### ISSUE TITLE
Religious Pressure Missing

### ISSUE THREAD

**Issue by zozer** [LoneGazebo/Community-Patch-DLL#6101]
_1. Mod version:_
1/11

_2. Mod list:_
More Unique Components

_3. Error description:_ 
Religious Pressure is missing and/or not displaying.  
![image](https://user-images.githubusercontent.com/10717704/72227950-2c0ed400-3570-11ea-87dc-9534a0d10afe.png)
As seen here, neither religion is showing religious pressure despite one of them being mine and another should have at least some since neighboring cities have that religion.
_4. Steps to reproduce (optional):_
Make a religion and see.

**Comment by LoneGazebo** [comment]
@ilteroi related to open borders etc. issues?

**Comment by ilteroi** [comment]
nah. different pathfinder. but i'll check

**Comment by zozer** [comment]
I did notice that religious pressure was working in a select few cities, but the value is way lower that what it used to be (like my own religion getting minimal competition getting like +11)

**Comment by Stalker012345** [comment]
The pressure from TRs is working (which iteroi might be what your seeing), but the passive looks completely dead.

**Comment by silentreply** [comment]
I just opened another report with additional details, I hadn't seen this one first. https://github.com/LoneGazebo/Community-Patch-DLL/issues/6113#issue-549338318

From what I've gathered so far, only holy cities generate religious pressure, and only on themselves. The pressure from holy cities is not applied to other cities. Pressure modifiers do not appear to work. Trade routes are however creating pressure.
